packages/schema/lib/schemas/RequestSchema.js

```diff
@@ -74,6 +75,11 @@ module.exports = makeSchema(
           }
         },
         additionalProperties: false
+      },
+      serializeValueForCurlies: {
+        description:
+          'A function to customize how to serialize a value for curlies `{{var}}`` in the request object. By default, when this is unspecified, the request client only replaces curlies where variables are strings, and would throw an error for non-strings. The function should accepts a single argument as the value to be serialized and return the string representation of the argument.',
```

Comment:
there's an extra backtick on your `{{var}}` making the rest of this line highlight weird in markdown